components/src/CSS/css.story.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
```

Comment:
Do you think it would make sense to have a storybook in the css package? I like this here, but it also would make sense to have the "tests" in the same package as the code.

What do you think @matthewlyle ?

Reply:
After discussion with Matt we agreed to add storybook to the css workspace. I'm doing that.

Reply:
👍 I added a typography story in the `css` and removed this one.